doc/documentation_data_format.rst

```diff
@@ -157,14 +161,19 @@ Detailed field description
     concentration/amount of that species and will override the initial
     concentration/amount given in the SBML model or given by a preequilibration
     condition. If ``NaN`` is provided for a condition, the result of the
-    preequilibration (or initial concentration/amount from the SBML model, if
+    preequilibration (or initial concentration/amount from the model, if
     no preequilibration is defined) is used.
 
   - ``${compartmentId}``
 
     If a compartment ID is provided, it is interpreted as the initial
     compartment size.
 
+  - For all other entities, values are statically replaced at all time points.
```

Comment:
In discussions with @dweindl, we have identified that this explanation is insufficient for rules that assign rates (where we only want to replace initial values not the rate at all timepoints). I am proposing the following text instead:

```
For all other entities, the value in the condition table either replaces the initial value or the value at all timepoints based on whether the model entity has a rate law assigned or not. For model entities that have constant algebraic assignments (but not necessarily constant values), i.e, that do not have a rate of change with respect to time assigned and that are not subject to event assignments, the algebraic assignment is replaced statically at all timepoints. Examples for such model entities are the targets of SBML `AssignmentRules`. 

For all other entities, e.g., those that are assigned by SBML `RateRules`, only the initial value value can be assigned in the condition table. If an assignment of the rate of change with respect to time or event assignment is desired, the values of model entities that are used to define rate of change or event assignments must be assigned in the condition table. If no such model entities exist, assignment is not possible.
```

This text may, in fact ,replace the definition above and parameters/compartments/species could be listed as examples of the two categories.

Reply:
Thanks @FFroehlich. Looks good to me. 

@fbergmann : What do you think?

Reply:
@dweindl so it would mean, if the entity reference has an assignment rule, drop the assignment rule, right? That would essentially ensure that the value assigned holds for all timepoints. If that is what was meant, then its all good to me.

Reply:
Not really dropping the assignment rule, but rather replacing the `math` attribute of the `AssignmentRule` by the provided value. (Well, it's the same as dropping the `AssignmentRule` and replacing it by a different one with the same `variable`.)

Reply:
> Not really dropping the assignment rule, but rather replacing the value of the `math` attribute of the `AssignmentRule` by the provided value. (Well, it's the same as dropping the `AssignmentRule` and replacing it by a different one with the same `variable`.)

Reply:
@FFroehlich I included your suggestion. Could you please have another look?